src/gccrs/config.rs

```diff
@@ -48,28 +48,21 @@ impl DumpedOption {
             _ => Err(invalid_input),
         }
     }
+}
 
-    // FIXME: Should we use the fmt::Display trait?
+impl Display for DumpedOption {
```

Comment:
NIT (not blocking merging): Any reason why this is called `DumpedOption` instead of just `Option`?

Reply:
`Option` whould shadow `std::option::Option`.

Reply:
I didn't want to call it `Option` as I thought it would be confusing for the reason bjorn evoked, and I thought that it was an okay-ish name since the the options were retrieved using `-frust-dump-target_options`